css/templates/lune/pages/_landing.css

```diff
@@ -127,8 +132,18 @@
       height: calc(var(--lune-landing-page-height) / 0.99);
     }
 
-    .lune-landing-wrapper {
+    .lune-landing-content {
```

Comment:
Not really, as the position fixed is applied inside a @support query. If the browser doesn't support clip-path, those element have another value for their position